ledger-tool/src/main.rs

```diff
@@ -1882,7 +1882,20 @@ fn main() {
                     .long("no-compaction")
                     .required(false)
                     .takes_value(false)
-                    .help("Skip ledger compaction after purge")
+                    .help("--no-compaction is deprecated, ledger compaction \
```

Comment:
`--help` update is never gonna be seen 'cause `.hidden(true)` :wink:

Reply:
True, master has the `.hidden(hidden_unless_forced())` which is not apparent here, but good point that no need to update the arg help in the backport.